### PR TITLE
[NCL-3177] Add view and download button to alignment

### PR DESCRIPTION
**Pull request by thescouser89**

### CODE REVIEW

**Comment by thescouser89** [comment]
@thescouser89: Don't forget to backport to v1.2

**Comment by thescouser89** [comment]
retest this please

File: ui/app/build-records/RecordRepourResultController.js [hunk 1]
```diff
@@ -20,7 +20,12 @@
 
   angular.module('pnc.build-records').controller('RecordRepourResultController', [
     'repourLog',
-    function(repourLog) {
+    'REST_BASE_URL',
+    'BUILD_RECORD_ENDPOINT',
+    'recordDetail',
+    function(repourLog, REST_BASE_URL, BUILD_RECORD_ENDPOINT, recordDetail) {
+      this.logUrl = REST_BASE_URL + BUILD_RECORD_ENDPOINT.replace(':recordId', recordDetail.id) + '/repour-log';
+      this.logFileName = recordDetail.id + '_' + recordDetail.buildConfigurationName + '_' + recordDetail.status + 'repour-log.txt';
```
Comment (matedo1) [inline ui/app/build-records/RecordRepourResultController.js]: You could add one more underscore repour-log.txt -> _repour-log.txt

Comment (thescouser89) [reply ui/app/build-records/RecordRepourResultController.js]: Good eye! :) Wi do the chnge

**Comment by thescouser89** [comment]
retest this please

**Comment by thescouser89** [comment]
retest this please

**Comment by thescouser89** [comment]
retest this please

**Comment by thescouser89** [comment]
retest this please

**Comment by thescouser89** [comment]
retest this please

**Pull request merged**